roles/matrix-bridge-appservice-slack/templates/systemd/matrix-appservice-slack.service.j2

```diff
@@ -0,0 +1,38 @@
+#jinja2: lstrip_blocks: "True"
+[Unit]
+Description=Matrix Appservice Slack server
+{% for service in matrix_appservice_slack_systemd_required_services_list %}
+Requires={{ service }}
+After={{ service }}
+{% endfor %}
+{% for service in matrix_appservice_slack_systemd_wanted_services_list %}
+Wants={{ service }}
+{% endfor %}
+
+[Service]
+Type=simple
+ExecStartPre=-/usr/bin/docker kill matrix-appservice-slack
+ExecStartPre=-/usr/bin/docker rm matrix-appservice-slack
+
+# Intentional delay, so that the homeserver (we likely depend on) can manage to start.
+ExecStartPre=/bin/sleep 5
+
+ExecStart=/usr/bin/docker run --rm --name matrix-appservice-slack \
+			--log-driver=none \
+			--network={{ matrix_docker_network }} \
+			-v {{ matrix_appservice_slack_config_path }}:/config:z \
+			-v {{ matrix_appservice_slack_data_path }}:/data:z \
```

Comment:
Maybe we need an extra thing here, to handle the case where matrix-nginx-proxy is disabled.

Example:

```

			{% if matrix_appservice_slack_container_http_host_bind_port %}
			-p {{ matrix_appservice_slack_container_http_host_bind_port }}:{{ matrix_appservice_slack_slack_port }} \
			{% endif %}
```

.. which necessitates the following new variable in `defaults/main.yml`:

```yaml
# Controls whether the appservice-slack container exposes its HTTP port (tcp/9003 in the container).
#
# Takes an "<ip>:<port>" or "<port>" value (e.g. "127.0.0.1:9003"), or empty string to not expose.
matrix_appservice_slack_container_http_host_bind_port: ''
```

and the following definition in `group_vars/matrix_servers`:

```yaml
matrix_appservice_slack_container_http_host_bind_port: "{{ '' if matrix_nginx_proxy_enabled else '127.0.0.1:{{ matrix_appservice_slack_slack_port }}' }}"
```

-------

I see that you already have a `matrix_appservice_slack_container_http_host_bind_port` variable in `defaults/main.yml`, but its comment block seems inaccurate.